dockerfiles/compile.go

```diff
@@ -124,8 +124,8 @@ var dockerfileArguments = []templateArguments{
 		install_packages: `RUN set -x; apt-get update && \
 		DEBIAN_FRONTEND=noninteractive apt-get -y install git systemd \
 		autoconf libtool libcurl4-openssl-dev libltdl-dev libssl-dev libyajl-dev \
-		build-essential cmake bison flex file libsystemd-dev \
-		devscripts cdbs pkg-config openjdk-${OPENJDK_MAJOR_VERSION}-jdk zip`,
+		build-essential bison flex file libsystemd-dev \
+		devscripts cdbs pkg-config openjdk-${OPENJDK_MAJOR_VERSION}-jdk zip` + installCMake,
```

Comment:
Shouldn't we only need `installCMake` for distros that ship < 3.20.0? Bookworm ships 3.25.1

Reply:
Yes, we should only use `installCmake` when we need to. I conflated all Ubuntu/Debian failures to this reason initially. Removed `installCmake`. Done!